examples/ciao/README.md

```diff
@@ -5,12 +5,18 @@ This is an example of a playbook to deploy CIAO using ansible.
 ## Prework
 
 ### Access
-Ansible requires that the user running the playbook has passwordless ssh access to the managed nodes, sudo privileges and python installed on each node.
+Ansible requires that the user running the playbook has passwordless ssh access
+and sudo privileges in the managed nodes and deployment machine.
 
 ### Requirements
 
 #### CIAO nodes
-CIAO can be installed in ClearLinux, Fedora 24 and Ubuntu 16.04 and its dependencies will be installed automatically.
+* Ansible requirements for managed nodes can be found
+[here](http://docs.ansible.com/ansible/intro_installation.html#managed-node-requirements)
+
+* CIAO can be installed in ClearLinux, Fedora 24 and Ubuntu 16.04. Its dependencies
+will be installed automatically
+
 
```

Comment:
Right now we still need to manually install docker I think.  I'm not sure how long it will be before we fix this.  Not sure if we want to mention that here.